data-raw/model-parameters.R

```diff
@@ -272,10 +150,6 @@ params <- list(
   ..surv_juv_delta_contact_points = (.0358 * -0.189),
   ..surv_juv_delta_total_diverted = (.5 * -0.0021),
```

Comment:
for any of these parameters that are number * another number, turns out that one value is from literature and the other is from calibration. we need to separate them into two parameters and update the submodels and the model to reflect this change. I'm not sure which number is the calibrated value but I bet the J+A paper or supplemental will have that information.